### PR TITLE
Update: CFDA Loader

### PR DESCRIPTION
**Pull request by dpb-bah**
**High level description:**
Same changes to the CFDA loader as the Broker.

**Technical details:**
* Pulls from GSA Bucket instead of FTP
* Fixes column change with additional comma
* Fixes the reformatting of the published and archive dates

**Link to JIRA Ticket:**
[DEV-1052](https://federal-spending-transparency.atlassian.net/browse/DEV-1052)

**The following are ALL required for the PR to be merged:**
- [x] Backend review completed
- [x] Unit & integration tests updated with relevant test cases
- Matview impact assessment completed (N/A)
- Frontend impact assessment completed (N/A)
- [x] Data validation completed
- API Performance evaluation completed (N/A)
- [x] Operations Team Review
This requires a couple new settings in `settings.py`:
```
CFDA_BUCKET_NAME = "falextracts"
CFDA_REGION = "us-east-1"
CFDA_FILE_PATH = "Assistance Listings/usaspendinggov/%Y/%m-%b/AssistanceListings_USASpendingGov_PUBLIC_WEEKLY_%Y%m%d.csv"
```

### CODE REVIEW

File: usaspending_api/settings.py [hunk 1]
```diff
@@ -53,6 +53,9 @@
 MONTHLY_DOWNLOAD_S3_BUCKET_NAME = ""
 BROKER_AGENCY_BUCKET_NAME = ""
 FPDS_BUCKET_NAME = ""
+CFDA_BUCKET_NAME = ""
```
[the remark this answers is not in the document]
Comment (kwhickey) [reply usaspending_api/settings.py]: Do you mean that a file in `usaspending-config` needs to be updated in conjunction with these changes? If so, @dpb-bah - make that change in the `usaspending-config` repo, link it to this PR (put the hyperlink of each PR in the other's description), and submit that PR for ops review.

Better to do the change and have it reviewed then just assume someone else will pick up the dependent work.
CC: @willkjackson, @nmonga91

**Comment by kwhickey** [comment]
Adding @sharb as a reviewer. @sharb - Can you run through the code here to see if there is anything majorly different in logic between the broker CFDA load and this (usaspending website) CFDA load. While they may not need all the same data, I think it's coming form the same source, yes? And if so, should generally handle the file-pickup in the same way.

**Comment by sharb** [comment]
@kwhickey In response to your comment, yeah it looks like it's using the same logic as broker cfda load to find the latest file from the same GSA bucket.

File: usaspending_api/references/management/commands/loadcfda.py [hunk 2]
```diff
@@ -1,21 +1,66 @@
-from django.core.management.base import BaseCommand
```
[the remark this answers is not in the document]
Comment (dpb-bah) [reply usaspending_api/references/management/commands/loadcfda.py]: Agreed. Codewise, we'd just need to update `load_reference_data` as well. Not sure if there are other scripts that call this though @willkjackson @sharb @hsharma96 .

File: usaspending_api/references/management/commands/loadcfda.py [hunk 3]
```diff
@@ -92,7 +139,7 @@ def load_cfda(abs_path):
                 cfda_program.program_accomplishments = row['Program Accomplishments (130)']
                 cfda_program.regulations_guidelines_and_literature = row['Regulations, Guidelines, '
                                                                          'and Literature (140)']
-                cfda_program.regional_or_local_office = row['Regional or Local Office (151) ']
+                cfda_program.regional_or_local_office = row['Regional or, Local Office (151)']
```
[the remark this answers is not in the document]
Comment (dpb-bah) [reply usaspending_api/references/management/commands/loadcfda.py]: I went through all the columns from the new CFDA and can confirm this was the only column that changed. The other change was the format for the Published Date/Archived Date which has been fixed as well. As for the offline columns changes, I wouldn't opposite it though I don't know how often the columns will change.

**Comment by dpb-bah** [comment]
Per OPs request:
* https://github.com/fedspendingtransparency/usaspending-config/pull/224
* https://github.com/fedspendingtransparency/data-act-build-tools/pull/148

**Pull request merged**